public/app/features/plugins/admin/hooks/usePluginInfo.tsx

```diff
@@ -31,6 +31,11 @@ export const usePluginInfo = (plugin?: CatalogPlugin): PageInfoItem[] => {
         label: t('plugins.details.labels.version', 'Version'),
         value: 'Managed by Grafana',
       });
+    } else if (plugin.isPreinstalled.withVersion) {
+      info.push({
+        label: 'Version',
+        value: 'Bundled with Grafana - ' + version,
```

Comment:
I wonder if it's better to keep it close to the managed plugins feature and just stick with `Bundled with Grafana`. Version history (though it's not very obvious to the user maybe) is still available for the user to see what is installed if they need to. cc @sympatheticmoose

Reply:
FWIW, my reasoning here is that for managed plugins people don't really need to care about the version since they cannot do anything about it (and it will usually be the latest?) but in this case, they may want to upgrade and they need to know which version they have.

FTR, I didn't remember that the Version history tells you in which version you are :)